gcp_variant_transforms/beam_io/vcf_parser.py

```diff
@@ -614,10 +614,10 @@ def _lookup_encoded_sample_name(self, sample_name):
     sample_id = self._encoded_sample_names.get(sample_name)
     if not sample_id:
       if self._sample_name_encoding == SampleNameEncoding.WITH_FILE_PATH:
-        sample_id = hex(hashing_util.generate_sample_id(
-            sample_name, self._file_name))
+        sample_id = hashing_util.generate_sample_id(
+            sample_name, self._file_name)
```

Comment:
nit: break line between the two input arguments, this way it will be easier to compare to the `else` section that only `sample_name` is being sent.

Reply:
Done.